packages/canvas-tokens-docs/components/DeprecatedColorGrid.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import {TokenGrid, formatJSVar} from './TokenGrid';
+import {formatName} from './ColorGrid';
+
+const sortMap: Record<string, number> = {
+  softer: 0,
+  soft: 1,
+  default: 2,
+  strong: 3,
+  stronger: 4,
+};
+
+export function sortSystemColorPalette(a: DeprecatedColorSwatch, b: DeprecatedColorSwatch) {
+  const aLevel = a.cssVar.split('-').at(-1) || '';
+  const bLevel = b.cssVar.split('-').at(-1) || '';
+  const first = aLevel in sortMap ? sortMap[aLevel] : Infinity;
+  const second = bLevel in sortMap ? sortMap[bLevel] : Infinity;
+  return first - second;
+}
+
+export function buildPalette(prefix: string, tokens: Record<string, string>) {
+  return Object.entries(tokens).map(([value, varName]) =>
+    buildDeprecatedColorSwatch(varName, `${prefix}.${value}`)
+  );
+}
+
+export function buildPaletteGroup(
+  prefix: string,
+  tokens: object,
+  sortFn?: (a: DeprecatedColorSwatch, b: DeprecatedColorSwatch) => number
+) {
+  return Object.entries(tokens)
+    .map(([key, value]) => {
```

Comment:
You should be able to use [flatMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap) here instead of `.map().flat()`.
```suggestion
    .flatMap(([key, value]) => {
```